bundles/org.openhab.transform.basicprofiles/src/main/java/org/openhab/transform/basicprofiles/internal/profiles/StateFilterProfile.java

```diff
@@ -553,6 +551,22 @@ public FunctionType(Function type, Optional<Integer> windowSize) {
             };
         }
 
+        public boolean alwaysAccept() {
+            if ((type == Function.DELTA || type == Function.DELTA_PERCENT) && acceptedState == UnDefType.UNDEF) {
+                return true;
+            }
+            if (type == Function.DELTA_PERCENT) {
+                // avoid division by zero
+                if (acceptedState instanceof QuantityType base) {
+                    return base.toBigDecimal().compareTo(BigDecimal.ZERO) == 0;
```

Comment:
The idea is to return true when the state is 0 ?

Reply:
Yes. I don't know what else to do, so I'm open to ideas.

Reply:
Thanks for providing the text. I've added it.